### ISSUE TITLE
Debug logging while debugging is disabled

### ISSUE THREAD

**Issue by milo526** [homebridge/homebridge#2698]
In my [homebridge-tuya-web](https://github.com/milo526/homebridge-tuya-web) plugin I created some [helper functions to aid in debugging](https://github.com/milo526/homebridge-tuya-web/blob/master/src/accessories/BaseAccessory.ts#L264).

These helper functions make use of the `platform.log.log(level, message, args)` call.
It turns out that calling `platform.log.debug(msg)` [properly disables itself](https://github.com/homebridge/homebridge/blob/0849c7711eb8dd7acc421f65e633bf98813d2aec/src/logger.ts#L143) when debugging is disabled, [the same however does not apply](https://github.com/homebridge/homebridge/blob/0849c7711eb8dd7acc421f65e633bf98813d2aec/src/logger.ts#L161) to `platform.log.log('debug', message, args)`

```javascript
platform.log.debug("This does not get printed with debug disabled");
platform.log.log("debug", "This does get printed with debug disabled.")
```

If this is not by design, I would like to create a PR to fix this.  
If this is by design I will alter my plugin to use the debug call directly.

**Comment by Supereg** [comment]
Thanks for pointing this out. Will be fixed with the next release.

**Comment by Supereg** [comment]
Fix was added to the beta channel and released as `v1.3.0-beta.6`. Thanks for reporting.
